sampler_remote.go

```diff
@@ -62,7 +62,7 @@ type SamplerUpdater interface {
 // delegates to it for sampling decisions.
 type RemotelyControlledSampler struct {
 	// These fields must be first in the struct because `sync/atomic` expects 64-bit alignment.
-	// Cf. https://github.com/uber/jaeger-client-go/issues/155, https://goo.gl/zW7dgq
+	// Cf. https://github.com/jaegertracing/jaeger-client-go/v3/issues/155, https://goo.gl/zW7dgq
```

Comment:
Please watch for these changes, v3 is incorrect here

Reply:
fixed them